README.md

```diff
@@ -42,7 +42,7 @@ Our research into this project stretches far and wide, covering a multitude of l
 
 # Tech Stack
 ![MongoDB](https://img.shields.io/badge/MongoDB-%234ea94b.svg?style=for-the-badge&logo=mongodb&logoColor=white)
-![Python](https://img.shields.♠io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
+![Python](https://img.shields.io/badge/Python-14354C?style=for-the-badge&logo=python&logoColor=white)
```

Comment:
This change looks fine, but also found that just removing the ♠ icon from the url resulted in a working python image. The two images are different in their color schemes , though that may not matter.